R4ML/R/java.bridge.R

```diff
@@ -38,10 +38,10 @@ log4j.Logger <- setRefClass("log4j.Logger",
     initialize = function(name = "org") {
       env <<- new.env()
       createLogger <- function(level) {
-        level_jref <- SparkR:::callJStatic("org.apache.log4j.Level", "toLevel", level)
+        level_jref <- sparkR.callJStatic("org.apache.log4j.Level", "toLevel", level)
```

Comment:
We have decided to use `SparkR::` for all our SparkR functions. Can you please change this to `SparkR::sparkR.callJStatic`?